draft-ietf-httpapi-ratelimit-headers.md

```diff
@@ -294,6 +294,53 @@ This example shows a 300MB remaining quota for an application in the next 60 sec
    RateLimit: "default";r=300000000;t=60;pk=:QXBwLTk5OQ==:
 ~~~
 
+# Problem Types (#problem-types)
+
+## Quota Exceeded
+
+This section defines the "https://iana.org/assignments/http-problem-types#quota-exceeded" problem type. A server MAY use this problem type if it wants to communicate to the client that the requests sent by the client exceed one or more Quota Policies. This problem type defines the extension member "violated-policies" as an array of strings.
```

Comment:
The name of the extension member is self-describing, but it might be useful to explicitly state what its value should be, for example:

```suggestion
This section defines the "https://iana.org/assignments/http-problem-types#quota-exceeded" problem type. A server MAY use this problem type if it wants to communicate to the client that the requests sent by the client exceed one or more Quota Policies. This problem type defines the extension member "violated-policies" as an array of strings, whose value is the names of the exceeded quotas.
```